test/integration/test_integration.rb

```diff
@@ -128,6 +128,7 @@ def test_create_call_with_amd_and_get_call_state
         machine_detection.delay_result = true
         machine_detection.callback_url = BASE_CALLBACK_URL + '/machineDetection'
         machine_detection.callback_method = 'POST'
+        machine_detection.machine_speech_end_threshold = 5.0
```

Comment:
```suggestion
        machine_detection.machine_speech_end_threshold = 6.2
```
I think the backend defaults this to 5 - can we set some random value in the test so that if we need to look through logs we will know our random value was set properly?